va/va_dec_av1.h

```diff
@@ -444,7 +444,8 @@ typedef struct  _VADecPictureParameterBufferAV1
     } pic_info_fields;
 
     /** \brief Supper resolution scale denominator.
-     *  value range [9..16]
+     *  When use_superres=1, value range [9..16].
+     *  When use_superres=0, denominator value will be 8.
```

Comment:
I think this should be clarified in one of two ways:

1) Say that "When use_superres=0, superres_scale_denominator is ignored and the driver shall assume the denominator to be 8".

2) Say that "When use_superres=0, superres_scale_denominator must be 8".

(1) is how the iHD driver currently implements it per your comments. (2) would be safer in order to handle any other driver that might have implemented AV1 decoding already (if any).

Reply:
Thank you for the suggestion Andres. I will go with (2).